### PR TITLE
computers faker data generator and database builder features finished

### PR DESCRIPTION
**Pull request by electriceeli**
I added the computer-faker.js, computer.js in databases, and computers.json plus edited the scripts in package.json to include hyphens so they can be called with npm run

To test run 'npm run build-tables'

Associated Issues: #14, #17 

I think should be approved because the computer table and data is essential to MVP for our final API

:shipit:

### CODE REVIEW

**Review by hunterphillips — COMMENTED** [review]
👍

**Review by Valerie-Freeman — APPROVED** [review]
Formatting looks good, ran code and saw the computers table in DB Browser 👍 
Uncomment  commented out code

File: databases/computers.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+const sqlite3 = require('sqlite3').verbose();
+const db = new sqlite3.Database('bangazon.sqlite');
+const { readFileSync } = require('fs');
+const { computers } = JSON.parse(readFileSync("./data/computers.json"));
+
+module.exports.buildComputerTable = () => 
+  db.serialize(() => {
+    // db.run(`DROP TABLE IF EXISTS computers`);
```
Comment (melissabell456) [inline databases/computers.js]: please uncomment or remove if not necessary :)

**Review by melissabell456 — CHANGES_REQUESTED** [review]
please uncomment drop table

**Review by hunterphillips — APPROVED** [review]

**Pull request merged**